### ISSUE TITLE
How many epochs should I train to get a well performed TransFuser Model.

### ISSUE THREAD

**Issue by hk-zh** [autonomousvision/transfuser#141]
Dear Authors,
I am training the transfuser model with the '02_05_withheld' setting cause I would like to test the model in the town05 long benchmark. I trained for 30 epochs. However, the testing result shows that the agent can barely finish the task. It always runs out of the street. Could I ask how many epochs should I train normally? 
Also, I attach the args below:

{
  "id": "transfuser",
  "epochs": 41,
  "lr": 0.0001,
  "batch_size": 20,
  "logdir": "log/transfuser",
  "load_file": "log/transfuser/model_3.pth",
  "start_epoch": 0,
  "setting": "02_05_withheld",
  "root_dir": "/home/data2/data/data",
  "schedule": 1,
  "schedule_reduce_epoch_01": 30,
  "schedule_reduce_epoch_02": 40,
  "backbone": "transFuser",
  "image_architecture": "regnety_032",
  "lidar_architecture": "regnety_032",
  "use_velocity": 0,
  "n_layer": 4,
  "wp_only": 0,
  "use_target_point_image": 1,
  "use_point_pillars": 0,
  "parallel_training": 0,
  "val_every": 2,
  "no_bev_loss": 0,
  "sync_batch_norm": 0,
  "zero_redundancy_optimizer": 0,
  "use_disk_cache": 0
}

Best wishes
Hongkuan Zhou

**Comment by Kait0** [comment]
31 epochs is what we usually use, because the learning rate gets reduced at epoch 30.

**Comment by hk-zh** [comment]
The model I trained with the '02_05_withheld' setup for 30 epochs still can't complete the task (the RC is only 5%). The agent always hits the statics when turning. Could I ask if you could provide the model trained with '02_05_withheld'?
Best wishes
Hongkuan zhou

**Comment by hk-zh** [comment]
Hi, I've found the problem. Sorry for the duplicate issue. I should have gone through the closed issues carefully. It's the same as issue #110.
Best regards
Hongkuan Zhou
